src/sorcha/ephemeris/simulation_driver.py

```diff
@@ -30,6 +30,8 @@ def create_ephemeris(orbits_df, pointings_df, args, configs):
 
     if args.output_ephemeris_file and args.outpath:
         ephemeris_csv_filename = os.path.join(args.outpath, args.output_ephemeris_file)
+    else:
+        ephemeris_csv_filename = ""
```

Comment:
OK to merge, but in the future I'd recommend two things:
* use `None` to denote an "unused" variable
* it's better to initialize all variables that are supposed to exist at the beginning of the function. I.e., rather than an `else` statement here, around line ~29 you'd add: `ephemeris_csv_filename = None`. This makes it clear which variables are supposed to exist throughout the function and what their "default" values are.

Reply:
Yeah. You're right none would be better. 5 am brain. I've updated. Take a quick look